MSAL/test/unit/native_auth/public/MSALNativeAuthUserAccountResultTests.swift

```diff
@@ -46,25 +46,28 @@ class MSALNativeAuthUserAccountResultTests: XCTestCase {
 
         sut = MSALNativeAuthUserAccountResult(
             account: account!,
-            authTokens: MSALNativeAuthTokens(accessToken: accessToken, refreshToken: refreshToken, rawIdToken: rawIdToken),
+            rawIdToken: rawIdToken,
             configuration: MSALNativeAuthConfigStubs.configuration,
             cacheAccessor: cacheAccessorMock
         )
         try super.setUpWithError()
     }
 
     // MARK: Call delegate properly tests
-
-    func test_whenAccountAndTokenExist_itReturnsCorrectData() {
-        let expectation = expectation(description: "CredentialsController")
-        let authTokens = MSALNativeAuthUserAccountResultStub.authTokens
-        let mockDelegate = CredentialsDelegateSpy(expectation: expectation, expectedResult: MSALNativeAuthTokenResult(authTokens: authTokens))
-        mockDelegate.expectedAccessToken = authTokens.accessToken.accessToken
-        mockDelegate.expectedExpiresOn = authTokens.accessToken.expiresOn
-        mockDelegate.expectedScopes = authTokens.accessToken.scopes.array as? [String] ?? []
-        sut.getAccessToken(delegate: mockDelegate)
-        wait(for: [expectation], timeout: 1)
-    }
+    // To-do: change this test by moching MSAL cache accessor
+    
+//    func test_whenAccountAndTokenExist_itReturnsCorrectData() {
```

Comment:
Why is this method now commented? What we changed in the code in order to invalidate this test?

Reply:
More than mocking the CacheAccessor we should make "mockable" the "acquireTokenSilent" function. A possible way to do tha is this onet:
- Create a new interface with the "acquireTokenSilenty(configuration, challengeTypes, MSALSilentTokenParameters)
- Create a new wrapper class for the MSALNativeAuthPublicClientApplication that implements the interface created in the previous point
- Inject this dependency in the MSALNativeAuthUserAccountResult class (use the class created in the previous point as default value)
- Inject a spy version of the interface created at point 1 to check how the MSALNativeAuthUserAccountResult operates when custom error/result are returned to that class

Reply:
Ok, let's do it in another PR. Removing the commented code.